### PR TITLE
Extract code to methods

### PR DESCRIPTION
**Pull request by andrewbents**

### CODE REVIEW

File: forticlientsslvpn_autoconnect.py [hunk 1]
```diff
@@ -17,30 +17,48 @@
...
+    connection = poplib.POP3_SSL(popserver)
+    connection.user(email)
+    connection.pass_(emailpassword)
+    return connection
+
+def extractAuthCode(popConnection):
+    lines = popConnection.retr(len(M.list()[1]))[1] # last message
+    msg_content = b'\r\n'.join(lines).decode('utf-8')
+    msg = Parser().parsestr(msg_content)
+    code = msg.get_payload(decode=True)[34:40] # extract 6 digit code
```
Comment (wowshakhov) [inline forticlientsslvpn_autoconnect.py]: return code

File: forticlientsslvpn_autoconnect.py [hunk 2]
```diff
@@ -17,30 +17,48 @@
...
-child.sendline(code)
-child.expect('STATUS::Login succeed')
-child.logfile = sys.stdout
-child.expect('STATUS::Tunnel running')
-
-print route
-pexpect.run(route)
-
-child.interact()
+def loginToVPN(forticlient):
```
Comment (wowshakhov) [inline forticlientsslvpn_autoconnect.py]: convert to snake case

**Review by wowshakhov — CHANGES_REQUESTED** [review]

File: forticlientsslvpn_autoconnect.py (hunk 2 above)
Comment (andrewbents) [reply forticlientsslvpn_autoconnect.py]: *applicable elsewhere*

**Comment by andrewbents** [comment]
@wowshakhov can you check again?

**Review by wowshakhov — APPROVED** [review]

**Pull request merged**